applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/pm_static_ZDebug.yml

```diff
@@ -0,0 +1,41 @@
+app:
+  address: 0x10200
+  region: flash_primary
+  size: 0xe7e00
+mcuboot:
+  address: 0x0
+  region: flash_primary
+  size: 0x10000
```

Comment:
how much flash does the MCUBoot uses at the moment? Can this be reduced?

Reply:
FLASH uasge for MCUBoot is currently: `64856 B        64 KB     98.96%`.
I will take a look at possible optimizations to reduce FLASH usage (I already added some).

Reply:
I expect there is little to be done here as we have both external flash and serial recover that will add usb.

Reply:
I think there could be some options enabled by default that are specific to Thingy53 (e.g. some drivers). I already found a few looking at Kconfig. The memory optimizations will be added as a separate PR.